workbench-service/Dockerfile.aspire

```diff
@@ -0,0 +1,50 @@
+ARG python_image=python:3.11-slim
```

Comment:
can you please explain the differences between this dockerfile and the existing one? thanks

Reply:
Current dockerfiles use sshd, which conflicts with ports exposed by the running container when deployed to Azure Container Apps. Had to duplicate to leave current deployment working and create a different for Aspire.

Reply:
in our chat, we discussed getting rid of this file (by adding the SSH_ENABLED arg and exporting it for the docker-entrypoint script)? is that still possible?